openlibrary/catalog/author/merge.py

```diff
@@ -129,7 +129,7 @@ def do_normalize(author_key, best_key, authors):
             need_update = True
     else:
         best = ol.get(best_key)
-        author_keys = set(k for k in a.keys() + best.keys() if k not in ('key', 'last_modified', 'type', 'id', 'revision'))
+        author_keys = set(k for k in list(a.keys()) + list(best.keys()) if k not in ('key', 'last_modified', 'type', 'id', 'revision'))
```

Comment:
`.keys()` are not required.  The following are all equivalent.
```
for key in a:  # unless a could change during iteration
for key in list(a):
for key in list(a.keys()):
```
In this case, `list(a) + list(best)` would work just fine.